### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by robertpi**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/setup_overview/configure_data_security/_index.md [hunk 1]
```diff
@@ -19,6 +19,39 @@ The Datadog Agent and some tracing libraries have  options available to address
...
+*  `http.request.content_length` - The size of the request payload body in bytes.
+*  `http.request.content_length_uncompressed` - The size of the uncompressed request payload body after transport decoding.
+*  `http.request.headers.*` - The request http headers. None will be collected by default, users can opt-in configure them.
+*  `http.response.content_length` - The size of the response payload body in bytes.
+*  `http.response.content_length_uncompressed` - The size of the uncompressed request payload body after transport decoding.
+*  `http.response.headers.*` - The response http headers. None will be collected by default, users can opt-in configure them.
+
+Some libraries may not yet implement all of these tags. Please check your libraries release notes for their availability.
+
+### Configuring HTTP data
```
Comment (robertpi) [inline content/en/tracing/setup_overview/configure_data_security/_index.md:43]: wasn't sure if this next section should be a table.

File: content/en/tracing/setup_overview/configure_data_security/_index.md [hunk 2]
```diff
@@ -13,12 +13,52 @@ aliases:
...
+The performance data and traces that you're collecting with Datadog can contain sensitive information that you want to filter out, obfuscate, scrub, filter, modify, or just not collect. Additionally, it may contain synthetic traffic that might cause your error counts to be inaccurate or Datadog to not accurately indicate the health of your services.
 
-The Datadog Agent and some tracing libraries have  options available to address these situations and modify or discard spans, and various options are described below.  These docs cover several common methods for configuring Tracer and Agent to achieve these security requirements.
+The Datadog Agent and some tracing libraries have options available to address these situations and modify or discard spans, and various options are described below. These docs cover several common methods for configuring Tracer and Agent to achieve these security requirements.
 
 If your fine-tuning needs aren't covered and you need assistance, reach out to [the Datadog support team][1].
 
+## HTTP data collected
```
Comment (yoannrobin) [inline content/en/tracing/setup_overview/configure_data_security/_index.md:22]: I would move this section under the "Generalizing resource names and filtering baseline" section as one of the first questions are around PIIs and Ids in ressource names

File: content/en/tracing/setup_overview/configure_data_security/_index.md [hunk 3]
```diff
@@ -13,12 +13,52 @@ aliases:
...
+
+### Configuring a client IP header
+
+Datadog automatically attempts to resolve `http.client_ip` from a number of well known headers, such as `X-Forwarded-For`. If you use a custom header for this field, or want to bypass the resolution algorithm, set the `DD_TRACE_CLIENT_IP_HEADER` environment variable and the library looks only in the specified header for the client IP.
+
+If you do not wish to collect the client IP value because you consider it sensitive data, or for any other reason, set the `DD_TRACE_CLIENT_IP_HEADER_DISABLED` environment variable to `true`. It is `false` by default.
+
+### Redacting the query in the URL
+
+The `http.url` tag is assigned the full URL value, including the query string. The query string could contain sensitive data, so by default Datadog parses it and redacts suspicious-looking values. This redaction process is configurable. To modify the regular expression used for redaction, set the `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP` environment variable to a valid regex of your choice. Valid regex is platform-specific.
```
[the remark this answers is not in the document]
Comment (robertpi) [reply content/en/tracing/setup_overview/configure_data_security/_index.md]: Done, see 93cbce69747bc2c4e5ad4fd55e60d6057cda81e1

File: content/en/tracing/setup_overview/configure_data_security/_index.md (hunk 2 above)
Comment (robertpi) [reply content/en/tracing/setup_overview/configure_data_security/_index.md]: Done, see 93cbce69747bc2c4e5ad4fd55e60d6057cda81e1

File: content/en/tracing/setup_overview/configure_data_security/_index.md [hunk 4]
```diff
@@ -172,6 +172,46 @@ apm_config:
...
+
+### Configuring a client IP header
+
+Datadog automatically attempts to resolve `http.client_ip` from a number of well known headers, such as `X-Forwarded-For`. If you use a custom header for this field, or want to bypass the resolution algorithm, set the `DD_TRACE_CLIENT_IP_HEADER` environment variable and the library looks only in the specified header for the client IP.
+
+If you do not wish to collect the client IP value, set the `DD_TRACE_CLIENT_IP_HEADER_DISABLED` environment variable to `true`. It is `false` by default.
+
+### Redacting the query in the URL
+
+The `http.url` tag is assigned the full URL value, including the query string. The query string could contain sensitive data, so by default Datadog parses it and redacts suspicious-looking values. This redaction process is configurable. To modify the regular expression used for redaction, set the `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP` environment variable to a valid regex of your choice. Valid regex is platform-specific. When the regex find a suspicious key / value pair, it will replace it with `<redacted>`.
```
Comment (kayayarai) [inline content/en/tracing/setup_overview/configure_data_security/_index.md:203]:
```suggestion
The `http.url` tag is assigned the full URL value, including the query string. The query string could contain sensitive data, so by default Datadog parses it and redacts suspicious-looking values. This redaction process is configurable. To modify the regular expression used for redaction, set the `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP` environment variable to a valid regex of your choice. Valid regex is platform-specific. When the regex finds a suspicious key-value pair, it replaces it with `<redacted>`.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**